### ISSUE TITLE
fix_gopath.sh is broken since the addition of quotes

### ISSUE THREAD

**Issue by frobware** [kubernetes/autoscaler#1862]
Since #1837 no globbing will take place.

For example:

```console
aim@mb:~/go/src/k8s.io/autoscaler/cluster-autoscaler
$ ./fix_gopath.sh 
Updating /Users/aim/go
Updating staging dep api
Initialized empty Git repository in /Users/aim/go/src/k8s.io/api/.git/
cp: /Users/aim/go/src/k8s.io/kubernetes/staging/src/k8s.io/api/*: No such file or directory
```

Whereas if you revert PR #1837

```console
$ ./fix_gopath.sh 
Updating /Users/aim/go
Updating staging dep api
Initialized empty Git repository in /Users/aim/go/src/k8s.io/api/.git/
[master (root-commit) da99f9a] from_staging
 321 files changed, 210434 insertions(+)
 create mode 100644 BUILD
 create mode 100644 CONTRIBUTING.md
...
```

**Comment by mboersma** [comment]
@frobware was this closed by #1865?

**Comment by frobware** [comment]
Yes, although I never tested it as I reverted the commit at the time I was
trying to use it.
